server/eclipse-project/src/main/java/us/freeandfair/corla/util/BallotSequencer.java

```diff
@@ -42,27 +45,34 @@ private BallotSequencer() {
    * audit boards.
    *
    * @param cvrs input CVRs
-   * @return sorted, deduplicated list of CVR database IDs
+   * @return sorted, deduplicated list of CVRs
    */
-  public static List<Long>
+  public static List<CastVoteRecord>
```

Comment:
Changes in this file were done only to support returning `CastVoteRecord`s rather than `Long`s.